frontend/templates/product-list/sections/FilterableProductsSection/index.tsx

```diff
@@ -133,9 +137,21 @@ export function FilterableProductsSection({
                      <Flex mt="1.5" mb="3" wrap="wrap" align="flex-start">
                         <CurrentRefinements />
                      </Flex>
-                     <Divider borderColor="gray.300" opacity="1" />
                   </Collapse>
-                  <FacetsAccordion productList={productList} />
+                  <Divider
+                     display={{
+                        md:
+                           hasAnyVisibleFacet &&
+                           currentRefinements.items.length > 0
+                              ? 'block'
+                              : 'none',
+                     }}
+                     borderColor="gray.300"
+                     opacity="1"
+                  />
+                  <Collapse in={hasAnyVisibleFacet} animateOpacity>
```

Comment:
I could be wrong here, but I would have expected *this* boolean to be set to `hasCurrentRefinements`

Reply:
i think this is currently correct, because this is the table of facet options you can choose from.